packages/react-components/react-card/library/src/components/Card/Card.cy.tsx

```diff
@@ -12,6 +12,7 @@ import {
   cardHeaderClassNames,
   cardPreviewClassNames,
   CardPreview,
+  type CardOnSelectionChangeEvent,
```

Comment:
we cannot use this until we support TS >=3.9, lets keep consistency for now with `import type {}`